crates/sui-swarm-config/tests/snapshots/snapshot_tests__network_config_snapshot_matches.snap

```diff
@@ -112,6 +112,17 @@ validator_configs:
       max_txn_age_in_queue:
         secs: 1
         nanos: 0
+      overload_monitor_interval:
```

Comment:
I don't think this change is a breaking change for the operator because of the default used here: https://github.com/MystenLabs/sui/blob/ea5666b9c19141f34bb797d4b52c85b4e6a46441/crates/sui-config/src/node.rs#L167

Reply:
yes, this should be okay